src/main/java/io/fabric8/vertx/maven/plugin/mojos/Constants.java

```diff
@@ -124,4 +124,21 @@
     String VERTX_RUN_MODE_JAR = "jar";
 
 
+    /**
+     *
+     */
+    String WILDCARD_JSON_FILES = "*.json";
+    /**
+     *
+     */
+    String WILDCARD_YAML_FILES = "*.yaml";
+    /**
+     *
+     */
+    String WILDCARD_YML_FILES = "*.yml";
```

Comment:
As said in a previous comment, I would remove these and use a single `String[] DEFAULT_RESOURCE_INCLUDES` or so.

And tbh, personally I would avoid `Constant` classes or interfaces and define the constants where they are used. 
http://stackoverflow.com/questions/29382728/constant-interface-anti-pattern-clarification